product_docs/docs/migration_portal/4/04_mp_migrating_database/01_mp_schema_extraction/known_issues_notes.mdx

```diff
@@ -0,0 +1,44 @@
+ - [Data pump utility](#data-pump-utility) 
+
+## Migration Portal
+
+### Wrapped objects  
+
+Migration Portal can't assess wrapped objects. If you include them in the extracted DDL, they aren't loaded into Migration Portal and aren't included in the count of objects that are assessed. If you want to assess wrapped objects and migrate them to EDB Postgres Advanced Server, include unwrapped versions of the objects in the DDL file that you upload to Migration Portal. The recommended way of doing this is to replace the wrapped versions in the Oracle database with clear-text versions before performing the schema extraction. After performing the schema extraction, you can replace the objects with the wrapped versions.
+
+### File encoding
+
+Migration Portal recommends the `.SQL` output file be in the UTF-8 encoding format. If you upload a `.SQL` file with non-UTF-8 encoding, all the characters that aren't compatible with UTF-8 are converted to the replacement character ‘�’ in the output DDL.
+
+!!! Tip
+    You can manually convert the extracted file to the UTF-8 format by using the iconv utility on Linux or the LibIconv utility on Windows. For example, if your database character set is in Latin-1 (ISO-8859-1), you can convert the extracted file to the UTF-8 format, as follows:
+ 
+    `iconv -f iso-8859-1 -t UTF-8 sample.sql > sample_utf8.sql`
+
+### ALTER statements
+
+Except for ALTER TABLE and ALTER TRIGGER, Migration Portal does not process any other ALTER statements in the DDL.
+
+## EDB DDL Extractor
+
+-   The EDB DDL Extractor script doesn't extract objects restored using `Flashback` that still have names like `BIN$b54+4XlEYwPgUAB/AQBWwA==$0`. If you want to extract these objects, you must change the names of the objects and rerun the extraction process.
+-   The EDB DDL Extractor extracts `nologging` tables as normal tables. Once these tables are migrated to EDB Postgres Advanced Server, WAL log files are created.
+-   The EDB DDL Extractor extracts objects only with VALID status. For any objects that have INVALID status that you want Migration Portal to assess, first update them to VALID.
+-   The EDB DDL Extractor doesn't extract objects that were obfuscated using the Oracle wrap feature. As such, these objects aren't included in the set of DDL assessed by the Migration Portal. If you want to assess these objects and migrate them to EDB Postgres Advanced Server, replace the wrapped versions of the objects with nonwrapped versions. See [Wrapped objects](#wrapped-objects) for more information.
+-   The EDB DDL Extractor creates Global Temporary tables to store the schema names and their dependency information. These tables are dropped at the end of successful extraction.
+-   The EDB DDL Extractor script doesn't extract schemas whose name starts with `PG_` because PostgreSQL doesn't support it. If you want to extract these schemas, you must change the name of the schema before extraction.
+
+## Data pump utility
```

Comment:
As noted in earlier comment, this sub-section title should probably be changed to "Oracle Data Pump utilities".

Reply:
Done.